pkg/controller.v1/tensorflow/pod.go

```diff
@@ -36,7 +36,8 @@ const (
 	tfConfig = "TF_CONFIG"
 
 	// gang scheduler name.
-	gangSchedulerName = "kube-batch"
+	gangSchedulerName                = "volcano"
```

Comment:
are we switching to volcano? Do we want to keep it as flag?

Reply:
The docs here https://www.kubeflow.org/docs/other-guides/job-scheduling/ shows we are using volcano.

Reply:
Do you think we should make it a CLI flag?